metric/tract/mBIKEWK-accuracy.csv

```diff
@@ -1,61 +1,61 @@
-37063001806,3.988603989,0.726978998,0.626612606,0.625459897,0.573355818,0.545046489,0.463847203,0.46870692
-37063001807,0.439184524,0.385191526,0.357217903,0.366300366,0.37702373,0.364806867,0.3298409,0.329649021
-37063001808,4.359430605,0.513022889,0.561612157,0.498972703,0.441100156,0.393791985,0.342396777,0.311412347
-37063001809,2.778565353,0.533175355,0.50731125,0.560316414,0.508221226,0.529265255,0.512820513,0.495482367
-37063001900,7.585139319,0.973053892,0.873998543,0.836236934,0.708382527,0.667779633,0.612557427,0.556844548
-37063002007,0.914759416,0.737398768,0.820212792,0.8913057,1.221524901,2.002385623,1.993321781,1.323633747
-37063002008,6.396866841,0.847457627,0.769230769,0.740283775,0.764818356,0.738461538,0.745341615,0.761904762
-37063002009,4.827586207,0.846660395,0.643776824,1.348825954,1.539638385,1.005731823,1.174339921,1.420629961
-37063002013,4.068078041,0.522298112,0.491199345,0.70171583,0.602536342,0.581969731,0.527319478,0.55503223
-37063002015,0.607128282,0.720228089,1.02787282,1.059730937,0.935138945,0.937335931,0.889927209,0.643939394
-37063002016,3.632320237,0.620048226,0.532414657,0.551410963,0.541919031,0.546799614,0.548741123,0.540540541
-37063002017,5.087312387,4.189177238,4.352208968,4.041402592,3.261418914,0.478318211,0.632797895,1.010336654
-37063002018,2.427545207,0.45,0.394981413,0.368124729,0.365434222,0.34034034,0.305755396,0.308754086
-37063002019,0.85944415,1.185684165,0.412938747,0.440852314,0.447594181,0.431654676,0.423878488,0.484066156
-37063002020,0.496323633,1.131043393,1.254740384,1.339334152,1.152541625,0.96348807,0.458344567,0.411124547
-37063002021,4.016393443,0.506427737,0.852477376,0.646444827,1.142722382,1.001468944,0.879761239,0.475435816
-37063002022,3.45435319,0.492984452,1.162921861,1.03472995,0.772089758,0.7315352,0.602087262,0.429953422
-37063002023,5.561861521,0.732807215,0.673778776,0.687285223,0.638977636,0.583657588,0.632544665,0.49410379
-37063002024,1.062313909,1.301736898,0.739767783,0.568219125,0.423203013,0.42173158,0.568767779,0.620511613
-37063002025,3.039702233,0.501672241,0.48446851,0.48391688,1.490290987,1.105866488,1.07877649,1.046672438
-37063002026,3.651266766,0.6271777,0.596491228,0.604766987,0.547504026,0.510051005,0.504600772,0.464734828
-37063002027,0.740775694,0.951370516,0.717372495,0.674194015,0.241525891,0.283505973,0.306306306,0.280204384
-37063002028,2.773061686,0.485306012,0.431143799,0.395348837,0.363247863,0.339118292,0.315515961,0.256410256
-37063002100,3.359616044,0.575263663,0.549273021,0.549805951,0.539169045,0.549805951,0.594821554,0.55410691
-37063002200,1.124211697,1.957159204,1.869763397,1.645535754,1.489626937,1.462877027,2.459860437,2.59031547
-37063002300,36.02941176,4.779411765,3.053435115,3.498542274,3.75,3.448275862,3.448275862,3.100775194
-37063980100,408.3333333,24.07407407,29.26829268,50,,,,
\ No newline at end of file
+"id","y_2011","y_2012","y_2013","y_2014","y_2015","y_2016","y_2017","y_2018","y_2019"
```

Comment:
@BiSeye Can you double-check source of this accuracy table? We have all years (mostly) replaced here with new values when it should only be the newest year added/changed

Reply:
This is strange.  What I have in my local and on this branch is a version that is not all changed but just the 2019.  I'll try to push again.  These pushes are pulling in stuff ive already done in backwards ways.  frustrating but learning path i suppose

Reply:
i see what happened.  Bike to work and Bus to work accuracy tables got jammed up.  Reworking them both cause my local is jacked now ahah.  
but i think im gettin to a resolution